indexer/indexer.toml

```diff
@@ -20,3 +22,11 @@ port = 8080
 host = "127.0.0.1"
 port = 7300
 
+[logger]
```

Comment:
I think we're going with removing this from config? ffti if you're going to make undo change in a different PR

Reply:
Yea will remove